hackclub.com.yaml

```
@@ -3841,3 +3842,7 @@ clubdash:
   - ttl: 600
     type: CNAME
     value: a.selfhosted.hackclub.com.
+hackducky: #add hackducky
```

```suggestion
hackducky: 
```
Comments aren't needed tbh but I approved regardless.